src/IHaskell/IPython.hs

```diff
@@ -188,6 +188,7 @@ installLabextension = SH.shelly $ do
         SH.</> ("labextensions" :: SH.FilePath)
         SH.</> ("jupyterlab-ihaskell" :: SH.FilePath)
 
+  print $ "Installing kernel in folder: " <> jupyterlabIHaskellDir
```

Comment:
Can you use `putStrLn` here instead of `print`? Also `(++)` instead of `(<>)` for GHC 8.0.